meinberlin/apps/budgeting/assets/Pagination.jsx

```diff
@@ -31,17 +38,26 @@ export const Pagination = (props) => {
           onClick={onPaginate}
         />
 
-        {elidedRange.map((num, idx) => (
-          <PaginationButton
-            type="num"
-            key={getKeyStr(num, idx)}
-            isActive={num === currentPage}
-            pageIndex={num}
-            onClick={onPaginate}
-            isDisabled={num === '…'}
-          />
-        ))}
-
+        {width > MOBILE_BREAKPOINT
+          ? elidedRange.map((num, idx) => (
+            <PaginationButton
+              type="num"
+              key={getKeyStr(num, idx)}
+              isActive={num === currentPage}
+              pageIndex={num}
+              onClick={onPaginate}
+              isDisabled={num === '…'}
+            />
+          ))
+          : (
+            <PaginationButton
+              type="mobile"
+              key={getKeyStr(0, 0)}
+              pageIndex={currentPage}
+              pageCount={pageCount}
+              onClick={onPaginate}
+              isDisabled
```

Comment:
we should pass `false` here I think, otherwise the text is grey and hard to read